dashboard/app/controllers/lti_v1_controller.rb

```diff
@@ -282,6 +284,10 @@ def sync_course
       changed: {}
     }
 
+    # Instantiate accumulator variable for metric event metadata
+    total_students = 0
+    total_sections = 0
```

Comment:
NIT: I might prefer this to be a one-liner like `total_students, total_sections = 0`.